src/api/proto/vispb/vis.proto

```diff
@@ -308,6 +308,17 @@ message TimeseriesChart {
   Axis y_axis = 4;
 }
 
+// StatChart The visual spec for a basic stat chart. Shows the latest value in the table as a big
+// number with units.
+message StatChart {
+  message Stat {
+    // The (numeric) data field that will be used as the Y axis
+    string value = 1;
```

Comment:
Shouldn't this be a numeric type? Like a double or string? 
How does this work with semantic types?

Reply:
Same way it does in `message Timeseries` in the same file: it's the id of the field in the results. The `relation` gets used to then find the semantic type of that field, see `stat-chart.tsx` in this same PR.